### ISSUE TITLE
FR: Link to the Bug Reporting Guide

### ISSUE THREAD

**Issue by chainsawriot** [gmbecker/contributing_to_r_lesson#3]
https://github.com/gmbecker/contributing_to_r_lesson/blob/65221fb54c58bcc5dfbc24cb1e4f8ec3ded2d6b6/outline.md?plain=1#L512

Maybe it is a good idea to link to this as well: https://www.r-project.org/bugs.html

**Comment by mmaechler** [comment]
Yes, please.  It's the *only* "official"  (R core) statement on the issues,  and it is maintained, too, by R-core -- intended to be changed slowly/rarely.
